### PR TITLE
[crx-doc] Add new group redirect to extension docs

### LINKED ISSUE #2924: The link of Manifest File is missing in the documentation
⚠️ This is for issues with the https://developer.chrome.com site, not Chromium itself.
If you want to file a bug with Chromium (the open-source project behind Google Chrome and other browsers), please use https://crbug.com.

**Describe the bug**
On this [page](https://developer.chrome.com/docs/extensions/reference/runtime/#method-getManifest) I can see a link to the _manifest file_, but that link doesn't exist.

**The screenshot of link
image

Thanks,
Farzad

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #2924 #2798

Changes proposed in this pull request:

- Add another redirect group to fix broken links. The following are a few examples:

[/docs/extensions/messaging](https://developer.chrome.com/docs/extensions/messaging) should redirect
to: [/docs/extensions/mv3/messaging](https://developer.chrome.com/docs/extensions/mv3/messaging)

[/docs/extensions/manifest](https://developer.chrome.com/docs/extensions/manifest) should redirect to
[/docs/extensions/mv3/manifest](https://developer.chrome.com/docs/extensions/mv3/manifest)

### CODE REVIEW

**Comment by AmySteam** [comment]
@dotproto I can't get this redirect or any of the existing redirects to work in the Netlify Deploy Preview link.

I can demo this working code when we meet this week.

File: redirects.yaml
Comment (dotproto) [inline redirects.yaml:361]: I think we should also include MV2 docs in the fallback check. There are some pages that aren't in the MV3 doc set but are in the MV2 and apps one. In those cases, it would be best to load the "extensions" version of the page before falling back to the app docs.

```suggestion
    - /docs/extensions/mv3/...
    - /docs/extensions/mv2/manifest/...
    - /docs/extensions/mv2/...
```

**Review by dotproto — APPROVED** [review]
LGTM once the suggestion is merged.

Just thinking out loud here, but this PR makes me wish we had a tool that would crawl the docs and create a diffable list of the links on each page. 🤔

**Pull request merged**
